style.css

```diff
@@ -0,0 +1,309 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  header nav {
+    padding-top: 40px;
+    margin-right: 50px;
+    float: right;
+    font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+      "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+    display: flex;
+    font-variant: small-caps;
+    font-size: 25px;
+    font-weight: bold;
+    border-top-color: transparent;
+  }
+  header li:hover {
+    transform: scale(1.1);
+    font-weight: bolder;
+  }
+  header li {
+    list-style-type: none;
+    display: inline-block;
+    margin-left: 25px;
+  }
+  #about {
+    margin-bottom: 15%;
+  }
+  #about p {
+    font-size: 25px;
+    line-height: 1cm;
+  }
+  
+  a {
+    text-decoration: none;
+    color: black;
+  }
+  body {
+    background-image: url("./images/bg.jpg");
+    background-size: cover;
+    background-position: center;
+    background-blend-mode: soft-light;
+  }
+  .profilepic {
+    margin-top: 10%;
+    float: right;
+    position: relative;
+    margin-bottom: 15%;
+  }
+  .intro {
+    font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+    font-weight: bold;
+    font-size: 100px;
+    position: absolute;
+    justify-content: center;
+    margin: 0 auto;
+    top: 35%;
+    left: 22%;
+    background-color: transparent;
+    z-index: 999;
+  }
+  #intro1:hover ~ #intro2:after {
+    content: "Web";
+    font-family: cursive;
+    color: rgb(88, 135, 173);
+  }
+  #intro1:hover ~ #intro3:after {
+    content: "Developer";
+    font-family: cursive;
+    color: rgb(88, 135, 173);
+  }
+  #intro1:after {
+    content: "Hello.";
+  }
+  #intro1:hover:after {
+    content: "Front-end";
+    font-family: cursive;
+    color: rgb(88, 135, 173);
+  }
+  #intro2:after {
+    content: "I am";
+  }
+  #intro3:after {
+    content: "Madhuri";
+  }
+  #intro3:hover:after {
+    content: "Developer";
+    font-family: cursive;
+    color: rgb(88, 135, 173);
+  }
+  .content {
+    display: inline-block;
+  }
+  
+  .pagestyle {
+    max-width: 80%;
+    font-size: 20px;
+    font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+    color: black;
```

Comment:
Never use black for your font.  Try to add a little more grey in it, it's more pleasing to the eye
Something like 
#302f2f
Also adding some text shadow can help with contrast on dark backgrounds
text-shadow: 0 0 white;